recipes-kernel/linux/linux-hikey-aosp_4.4.bb

```diff
@@ -71,4 +71,6 @@ do_configure() {
     bbplain "Saving defconfig to:\n${B}/defconfig"
     oe_runmake -C ${B} savedefconfig
     cp -a ${B}/defconfig ${DEPLOY_DIR_IMAGE}
+    bbplain "Saving .config to:\n${B}/config"
+    cp -a ${B}/.config ${DEPLOY_DIR_IMAGE}/config
```

Comment:
It was requested in some of the bugs reported against LKFT builds. The full config can be retrieved from running board, but that is a bit cumbersome (not everyone involved has hikey board)